### ISSUE TITLE
Why is the loss in the progress bar different from the one logged manually

### ISSUE THREAD

**Issue by OldWheat** [Lightning-AI/lightning#15831]
I follow the tutorial Lightning in 15 minutes(https://pytorch-lightning.readthedocs.io/en/stable/starter/introduction.html#maximize-flexibility).

But I don't understand what the training pbar means. Why is loss different from self.log("train_loss", loss, prog_bar=True)?

I look forward to receiving answers soon. Thanks!
![图片](https://user-images.githubusercontent.com/64899054/204078269-2759e52a-2ed6-437a-81e5-2d654a989254.png)
![图片](https://user-images.githubusercontent.com/64899054/204078275-4a9a2fb8-8b57-4284-bf96-2da8e1e5da46.png)

cc @awaelchli

**Comment by awaelchli** [comment]
The difference is that the default logged loss "loss" is smoothed over a window of a few values, while the value logged by you is the exact one that was seen on each step (not smoothed).
